src/main/java/com/example/bookshop/service/impl/UserServiceImpl.java

```diff
@@ -26,6 +33,10 @@ public UserResponseDto register(UserRegistrationRequestDto requestDto)
             ));
         }
         User user = userMapper.toModel(requestDto);
+        user.setPassword(passwordEncoder.encode(user.getPassword()));
+        Role role = roleRepository.findByName(RoleName.USER)
+                .orElseThrow(() -> new RegistrationException("Role Not Found"));
```

Comment:
add RoleName.USER to message
```suggestion
                .orElseThrow(() -> new EntityNotFoundException("Role not found"));
```